nomad/fsm.go

```diff
@@ -266,6 +268,20 @@ func (n *nomadFSM) applyDrainUpdate(buf []byte, index uint64) interface{} {
 	return nil
 }
 
+func (n *nomadFSM) applyFreezeUpdate(buf []byte, index uint64) interface{} {
+	defer metrics.MeasureSince([]string{"nomad", "fsm", "node_freeze_update"}, time.Now())
```

Comment:
all metrics are lacking tagged metrics(?)

Reply:
there is still work going on around metrics AFAIK